web/client/epics/playback.js

```diff
@@ -39,10 +39,14 @@ const domainArgs = (getState, paginationOptions = {}) => {
         ...paginationOptions
     }];
 };
+const domainRangeIdentify = (start, end) => `${start}/${end}`;
```

Comment:
As I said in another comment, you have to move this transformation inside `domainArgs` function. 
Anyway if you want to keep a function to identify this operation, you must find a better name.
Checking [this](https://docs.geoserver.org/latest/en/user/services/wms/time.html#specifying-an-absolute-interval) probably a good name is something like `toAbsoluteInterval`